apps/src/music/blockly/setup.ts

```diff
@@ -37,19 +37,17 @@ import {BlockConfig} from './types';
  */
 export function setUpBlocklyForMusicLab() {
   backupFunctionDefinitons();
-  Blockly.Extensions.register(
+
+  safeRegisterExtension(
```

Comment:
I made the changes in this file initially to make sure that the new player's Blockly setup wouldn't interfere with doing the same thing for the main lab view. Later I realized that we don't actually need to set up Blockly for the exemplar player. 
Unless there are any particular concerns, I'd like to leave this improvement in since it will make it so we don't hit the error that an extension has already been registered.